tests/strands/test_strands_tracing.py

```diff
@@ -99,6 +97,32 @@ async def new_run_loop(self, message, invocation_state):
         assert len(get_traces()) == 1
 
 
+def run_tool(agent, tool_uses, cycle_trace, invocation_state):
+    try:
+        from strands.agent.agent import run_tool
+        from strands.tools.executor import run_tools
+
+        return run_tools(
+            handler=lambda tool_use: run_tool(agent, tool_use, invocation_state),
+            tool_uses=tool_uses,
+            event_loop_metrics=agent.event_loop_metrics,
+            invalid_tool_use_ids=[],
+            tool_results=[],
+            cycle_trace=cycle_trace,
+            parent_span=agent.trace_span,
+        )
+    except ImportError:
+        from strands.tools.executors import SequentialToolExecutor
+        return SequentialToolExecutor()._execute(
```

Comment:
We should not use a private method in a third-party library.

Reply:
Seemingly we don't have a better replacement. because it is only used in testing code, it should be fine.

Reply:
cc @joelrobin18 Let us know if there is any better approach here.